tests/pipelines/multi_asic_201911.yml

```diff
@@ -0,0 +1,117 @@
+# Starter pipeline
+
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+
+# Add steps that build, run tests, deploy, and more:
+
+# https://aka.ms/yaml
+
+pr:
+- master
+
+name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+resources:
+  repositories:
+  - repository: sonic-mgmt
+    type: github
+    name: Azure/sonic-mgmt
+    endpoint: build
+
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: dut
+    value: vlab-08
+  - name: tbname
+    value: vms-kvm-four-asic-t1-lag
+  - name: inventory
+    value: veos_vtb
+  - name: testbed_file
+    value: vtestbed.csv
+  - name: ptf_name
+    value: ptf_test
+  - name: tbtype
+    value: multi-asic-t1-lag
+
+  jobs:
+  - job: four_asic_vs_test 
+    displayName: "kvmmultiasictest"
+    timeoutInMinutes: 0
+    steps:
+
+    - script: |
+        sudo mkdir -p /data/sonic-vm/images
+        wget https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-multiasic-vs-image-201911/lastSuccessfulBuild/artifact/target/sonic-vs.img.gz
+        sudo cp -v ./sonic-vs.img.gz /data/sonic-vm/images/sonic-vs.img.gz
+        sudo gzip -fd /data/sonic-vm/images/sonic-vs.img.gz
+        username=$(id -un)
+        sudo chown -R $username.$username /data/sonic-vm
+        pushd  /data/sonic-mgmt
+        git remote update
+        git reset --hard origin/master
+        docker rm -f sonic-mgmt-masic || true
+        ./setup-container.sh -n sonic-mgmt-masic -d /data
+        sed -i s/use_own_value/${username}/ ansible/veos_vtb
+        echo aaa > ansible/password.txt
+        parent_dir="/data/sonic-mgmt"
+        docker exec sonic-mgmt-masic bash -c "find / -name ansible"
```

Comment:
What's the purpose of this step?

Reply:
Was not required, removed this.
I have added additional step to reboot the dut before starting the tests.
The multi-asic VS DUT seems to be stable after a reboot, so have added this which could be removed in future once the DUT is stable.

Reply:
sample test result: https://dev.azure.com/mssonic/build/_build/results?buildId=11746&view=logs&j=9edfe4bb-6ff9-5d54-84a0-38181090e645&t=e8814286-d068-5824-9202-504d4c05448a&l=635